cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -1151,14 +1152,59 @@ export function searchAnnotationsAsync(
     };
 }
 
+export function addTagAsync(
+    labelID: number,
+    frame: number,
+): ThunkAction<Promise<void>, {}, {}, AnyAction> {
+    return async (dispatch: ActionCreator<Dispatch>): Promise<void> => {
+        const {
+            canvas: {
+                instance: canvasInstance,
+            },
+            job: {
+                instance: jobInstance,
+            },
+        } = getStore().getState().annotation;
+
+        dispatch({
+            type: AnnotationActionTypes.ADD_TAG,
+            payload: {
+                labelID,
+                objectType: ObjectType.TAG,
+                activeControl: ActiveControl.CURSOR,
+            },
+        });
+
+        canvasInstance.cancel();
```

Comment:
Probably it would be better to call ``canvasInstance.cancel()`` before dispatch
And also you do not need ```activeControl: ActiveControl.CURSOR``` because ``canvasInstance.cancel()`` does it.